src/libraries/System.Drawing.Common/src/System.Drawing.Common.csproj

```diff
@@ -8,8 +8,12 @@
     <TargetFrameworks>$(NetCoreAppCurrent)-Windows_NT;$(NetCoreAppCurrent)-Unix;$(NetCoreAppCurrent)-Browser;netcoreapp3.0-Windows_NT;netcoreapp3.0-Unix</TargetFrameworks>
     <ExcludeCurrentNetCoreAppFromPackage>true</ExcludeCurrentNetCoreAppFromPackage>
     <Nullable>enable</Nullable>
+    <NoWarn>$(NoWarn);nullable</NoWarn>
```

Comment:
This should not be added.  Why is it needed?  If it's specific to browser, it should be conditioned on TargetsBrowser.

Reply:
Yeah. I believe it might be hitting issues with the `PNSE` generated source. Maybe we should disable the nullable warnings for PNSE assemblies in a central place instead. Something like:

https://github.com/dotnet/runtime/blob/3099ce6fcaebaac637e895dcfa7156cfbd66861e/src/libraries/Directory.Build.targets#L9

Reply:
Otherwise I get a lot of error like this:
```
/Users/admin/dev/runtime/artifacts/obj/System.Drawing.Common/net5.0-Browser-Release/System.Drawing.Common.notsupported.cs(1193,70): error CS8618: Non-nullable field 'Default' is uninitialized. Consider declaring the field as nullable. [/Users/admin/dev/runtime/src/libraries/System.Drawing.Common/src/System.Drawing.Common.csproj]
```

Reply:
Yeah you're hitting: https://github.com/dotnet/arcade/issues/4616

I would say let's disable nullable warnings for PNSE assemblies. 

Let's add an entry to `libraries/Directory.Build.targets` below the line I shared above, like:

```xml
<NoWarn Condition="'$(GeneratePlatformNotSupportedAssembly)' == 'true' or '$(GeneratePlatformNotSupportedAssemblyMessage)' != ''">$(NoWarn);nullable</NoWarn>
```

Reply:
cc: @ericstj

Reply:
Done